tests/acceptance/expected-failures-with-oc10-server.md

```diff
@@ -0,0 +1,8 @@
+## This file contains the scenarios from web tests that are currently expected to fail on oC10
+
+### [Media Viewer](https://github.com/owncloud/ocis/issues/1106)
+- <div class="text-green">[webUIPreview/imageMediaViewer.feature:81](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIPreview/imageMediaViewer.feature#L81)</div>
```

Comment:
The `div` stuff does not work in my IDE. I am not sure if that sort of thing is allowed in the `md` "standard".
Anyway, I don't think that we need to be so fancy.